File: Back/src/ProEventos.Domain/Lote.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProEventos.Domain
{
    public class Lote
    {
       public int ID { get; set; }
       public string Nome { get; set; }
       public decimal Preco { get; set; } 
       public DateTime? DateInicio { get; set; }
       public DateTime? DateFim { get; set; }
       public int Quantidade { get; set; }
       public int EventoId { get; set; }
       public Evento Evento { get; set; }
    }
}

File: Front/ProEventos-App/src/app/model/Lote.ts
import { Evento } from "./Evento";

export interface Lote {
  ID: number;
  Nome: string;
  Preco:number;
  DateInicio?: Date;
  DateFim?: Date;
  Quantidade:number;
  EventoId: number;
  Evento: Evento;
}
